javascript/modules/client/examples.md

```diff
@@ -90,6 +90,117 @@ for await (const step of steps) {
   }
 }
 
+```
+### Depositing ETH to a DAO
+
+Handles the flow of depositing the native EVM token to an Aragon DAO.
+
+
+
+```ts
+import {
+  Client,
+  Context,
+  DaoDepositSteps,
+  GasFeeEstimation,
+  IDepositParams,
+} from "@aragon/sdk-client";
+import { contextParams } from "../context";
+
+const context: Context = new Context(contextParams);
+const client: Client = new Client(context);
+const depositParams: IDepositParams = {
+  daoAddress: "0x1234567890123456789012345678901234567890",
+  amount: BigInt(10), // amount in wei
+  reference: "test deposit", // optional
+};
+
+// gas estimation
+const estimatedGas: GasFeeEstimation = await client.estimation.deposit(
+  depositParams,
+);
+console.log(estimatedGas.average);
+console.log(estimatedGas.max);
+
+const steps = client.methods.deposit(depositParams);
+for await (const step of steps) {
+  try {
+    switch (step.key) {
+      case DaoDepositSteps.DEPOSITING:
+        console.log(step.txHash); // 0xb1c14a49...3e8620b0f5832d61c
+        break;
+      case DaoDepositSteps.DONE:
+        console.log(step.amount); // 10n
+        break;
+    }
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+```
+### Depositing ERC20 tokens to a DAO
+
```

Comment:
Can we make sure that the 3 ticke are placed next to the code and not next to the markdown? Or at least, leave some extra new line?

Reply:
I would also advice performing a `.trim()` on the processed contents and then add the extra new lines to create at least one empty line between blocks